edge-apis/component.go

```diff
@@ -27,10 +26,6 @@ func NewComponents(api *url.URL, schemes []string) *Components {
 
 	httpTransport := &http.Transport{
 		Proxy: http.ProxyFromEnvironment,
-		DialContext: (&net.Dialer{
-			Timeout:   10 * time.Second,
-			KeepAlive: 10 * time.Second,
-		}).DialContext,
```

Comment:
I just re-ran my tests w/ this not set, and it worked just fine. I'm not sure why it failed the first time I tried it.